src/Security/Authorization/Core/src/AuthorizationPolicy.cs

```diff
@@ -108,7 +108,27 @@ public static AuthorizationPolicy Combine(IEnumerable<AuthorizationPolicy> polic
     /// A new <see cref="AuthorizationPolicy"/> which represents the combination of the
     /// authorization policies provided by the specified <paramref name="policyProvider"/>.
     /// </returns>
-    public static async Task<AuthorizationPolicy?> CombineAsync(IAuthorizationPolicyProvider policyProvider, IEnumerable<IAuthorizeData> authorizeData)
+    public static Task<AuthorizationPolicy?> CombineAsync(IAuthorizationPolicyProvider policyProvider,
+        IEnumerable<IAuthorizeData> authorizeData) => CombineAsync(policyProvider, authorizeData,
+            Enumerable.Empty<AuthorizationPolicy>(),
```

Comment:
I wonder if it's better to use `Array.Empty` here since it would benefit from a slightly faster `Enumerable.Any()`

Reply:
Only slightly in since `Enumerable.Empty<T>` is also optimized for the Count check.